parser-typechecker/src/Unison/Runtime/IOSource.hs

```diff
@@ -607,7 +607,7 @@ unique[ae4e05d8bede04825145db1a6a2222fdf2d890b3044d86fd4368f53b265de7f9] type Do
   = Svg Text
 
 -- ex: Doc2.term 'List.map
-Doc2.term : 'a -> Doc2.Term
+Doc2.term : ∀ a g1 g. '{g} a ->{g1} Doc2.Term
```

Comment:
Added a manual type signature here so it has an identical hash to the current definition. This also shows that inference is leaving around some stray ability variables.

Reply:
@pchiusano What, if anything, do we need to do on this line after tweaking the inference (or whatever you and @tstat  settled on doing, if not that)?

Reply:
Nothing, we can leave it alone. The pretty-printer will still elide these "useless" ability variables since there's a lot of code in the wild that has them (including this definition - the inferred type is what I added here).